Gordon360/Controllers/NewsController.cs

```diff
@@ -110,6 +112,27 @@ public ActionResult<IEnumerable<StudentNewsCategoryViewModel>> GetCategories()
             return Ok(result);
         }
 
+        [HttpGet]
+        [Route("unapproved")]
+        [StateYourBusiness(operation = Operation.UPDATE, resource = Resource.NEWS_APPROVAL)]
+        public ActionResult<IEnumerable<StudentNewsViewModel>> GetNewsUnapproved()
+        {
+            var result = _newsService.GetNewsUnapproved();
+            if (result == null)
+            {
+                return NotFound();
+            }
+            return Ok(result);
+        }
+
+        [HttpGet]
+        [Route("test-route")]
+        public ActionResult<IEnumerable<StudentNewsUploadViewModel>> TestRoute()
+        {
+            var result = _newsService.GetNewsUnapproved();
+            return Ok(result);
+        }
+
```

Comment:
Did you mean to push this test route?

Reply:
Right, I should remove this before the push. There's one more fix I need to work on in this pr. The image appears to be transparent when I click 'Edit' button on my personal unapproved news item in the Cropper. Once this got fixed, I will remove the `test-route` on my last commit.

Reply:
Generally you shouldn't commit things that are solely used for testing at all. It can clutter up the commit history when you merge your branch. I believe VSCode lets you stash ranges of lines of code and Visual Studio might allow this as well. That's nice for when you want to save your temporary changes.